wiz/pipeline.yaml

```
@@ -0,0 +1,10 @@
+steps:
+  - label: "Scan different Directory"
+    command: <command here>
+    env:
+    - WIZ_API_ID: "<your-id-goes-here>"
```

The build is failing because it looks like this is invalid env. Try this!

```suggestion
    WIZ_API_ID: "<your-id-goes-here>"
```

---

done